docs/modules/ROOT/pages/index.adoc

```diff
@@ -26,7 +26,7 @@ _This section is non-normative._
 
 === Scope
 
-This specification describes a _<<C2PA assertion>>_ that allows an _<<_actor,actor>>_ to document their relationship to a _<<C2PA asset>>_ produced by them or on their behalf independently from the _<<C2PA claim generator>>,_ and to allow _<<_c2pa_manifest_consumer,consumers>>_ of a _<<C2PA asset>>_ to independently verify that the received asset was in fact produced by the named _<<_actor,actor>>_ and has not been tampered with.
+This specification describes a _<<C2PA assertion>>_ that allows a _<<_named_actor,named actor>>_ to document their relationship to a _<<C2PA asset>>_ produced by them or on their behalf independently from the _<<C2PA claim generator>>,_ and to allow _<<_c2pa_manifest_consumer,consumers>>_ of a _<<C2PA asset>>_ to independently verify that the received asset was in fact produced by the _<<_named_actor,named actor>>_ and has not been tampered with.
```

Comment:
I think this is an area that needs discussion...

Since a C2PA Manifest Consumer, as defined by the C2PA spec, knows NOTHING about CAWG - there is no expectation that it would do ANYTHING with this assertion.  Instead, what you really need to define is something like a "CAWG-aware C2PA Manifest Consumer"...